liquibase-standard/src/main/java/liquibase/serializer/core/xml/XMLChangeLogSerializer.java

```diff
@@ -185,8 +193,26 @@ public Element createNode(LiquibaseSerializable object) {
         }
         Element node = currentChangeLogFileDOM.createElementNS(namespace, nodeName);
 
+        if (object instanceof ColumnConfig) {
+            ColumnConfig cc = (ColumnConfig) object;
+            if (cc.isNullValue() && !cc.hasDefaultValue() && !cc.hasSortOrder() && cc.getConstraints() == null) {
+                node.setAttribute("name", (String) object.getSerializableFieldValue("name"));
+
+                Object type = object.getSerializableFieldValue("type");
+
+                if (null != type) {
+                    node.setAttribute("type", type.toString());
+                } else {
```

Comment:
Something else I just noticed is that I don't think this `else` is accurate as it is. I think it should be a separate `IF` block comparing if a value field (similar to what we are getting in L201) is `null`, if it's then we set `node.setAttribute("value", "null");`

Reply:
I will check it, and perhaps add more tests.

Reply:
Actually, in the latest change, I onlt have to check if the ColumnConfig only contains the column name alone.
The `type` case is handled by means of `ColumnConfig#isSerializable(boolean)`.